### ISSUE TITLE
Learning Rate problem 

### ISSUE THREAD

**Issue by Mahmood1968** [dell-ai-engineering/BigDL-ImageProcessing-Examples#5]
The right  learning rate which set as 0.005 and 0.003 , and 0.01 all these values when plotted in tensor board shows unsmooth curve this is an indication of bad training and the loss function never decreased after every iteration and not converge to the local minima , **In fact , if the gradient decent implemented correctly the loss function should be decrease after each iteration**. Secondly , if the loss function is increase that means the gradient decent is not implemented correctly. However, I tried different values as mentioned above and not one of them show smooth curve. 
I think , there is some issues in the library or dataset not correctly preprocessed (which gray images )  that cause this problem or some parameters are nor setting in the right way . I hope you can investigate the issue and come with a solution . Here , good link to understand how to set the learning rate .Please look at  this image for my code which never improved with any values of learning rate 
![learning_rate_not_smooth](https://user-images.githubusercontent.com/24418838/46901515-3d53c700-ce7b-11e8-85e6-a9f236178f12.PNG)

There is another good resource here 
https://www.coursera.org/lecture/machine-learning/gradient-descent-in-practice-ii-learning-rate-3iawu

**Comment by hhbyyh** [comment]
![image](https://user-images.githubusercontent.com/7981698/47039293-7d939d80-d138-11e8-9a4a-98a1917082d2.png)
Please try to adjust smoothing.
